deepmd/pt/utils/tabulate.py

```
@@ -0,0 +1,920 @@
+    def _layer_1(self, x, w, b):
+        w = torch.from_numpy(w)
+        b = torch.from_numpy(b)
+        t = torch.cat([x, x], dim=1)
+        return t, self.activation_fn(torch.matmul(x, w) + b) + t
+    
+    # Change the embedding net range to sw / min_nbor_dist
+    def _get_env_mat_range(self, min_nbor_dist):
+        sw = self._spline5_switch(min_nbor_dist, self.rcut_smth, self.rcut)
+        if isinstance(self.descrpt, deepmd.pt.model.descriptor.DescrptSeA) or isinstance(
+            self.descrpt, deepmd.pt.model.descriptor.DescrptDPA1
+        ):
```

Ruff

Multiple `isinstance` calls for expression, merge into a single call

Merge `isinstance` calls

(SIM101)